### PR TITLE
[Emiri-sage]- Interacting with the Celo Blockchain Using Web3.js A Beginners Guide 

### PR DESCRIPTION
**Pull request by emiridbest**
## Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

### CODE REVIEW

File: src/data-tutorials/users.tsx [hunk 1]
```diff
@@ -883,6 +883,14 @@ const Users: User[] = [
     website: 'blog/tutorials/how-to-connect-to-a-node-on-celo-using-docker',
     tags: ['celosage', 'celo', 'beginner', 'cli'], 
   },
+  
+{
+    title: 'Interacting with the Celo Blockchain Using Web3.js A Beginners Guide - A Voting App',
+    description: 'This tutorial teaches how to interact with smart contracts using web3.js simplified by Celo Contract Kit',
+    preview: require('./showcase/beginner/interacting-with-the-Celo-Blockchain-Using-Web3js-A-Beginners-Guide-A-Voting-App.png'),
+    website: 'blog/tutorials/how-to-connect-to-a-node-on-celo-using-docker',
```
Comment (developerkunal) [inline src/data-tutorials/users.tsx:891]:
```suggestion
    website: 'blog/tutorials/interacting-with-the-celo-blockchain-using-web3js-a-beginners-guide-a-voting-appr',
```

File: src/data-tutorials/users.tsx [hunk 2]
```diff
@@ -883,6 +883,14 @@ const Users: User[] = [
     website: 'blog/tutorials/how-to-connect-to-a-node-on-celo-using-docker',
     tags: ['celosage', 'celo', 'beginner', 'cli'], 
   },
+  
+{
+    title: 'Interacting with the Celo Blockchain Using Web3.js A Beginners Guide - A Voting App',
+    description: 'This tutorial teaches how to interact with smart contracts using web3.js simplified by Celo Contract Kit',
+    preview: require('./showcase/beginner/interacting-with-the-celo-blockchain-using-web3js-a-beginners-guide-a-voting-app.png'),
```
Comment (developerkunal) [inline src/data-tutorials/users.tsx:890]:
```suggestion
    preview: require('./showcase/beginner/interacting-with-the-Celo-Blockchain-Using-Web3js-A-Beginners-Guide-A-Voting-App.png'),
```

File: src/data-tutorials/users.tsx [hunk 3]
```diff
@@ -883,6 +883,14 @@ const Users: User[] = [
     website: 'blog/tutorials/how-to-connect-to-a-node-on-celo-using-docker',
     tags: ['celosage', 'celo', 'beginner', 'cli'], 
   },
+  
+{
+    title: 'Interacting with the Celo Blockchain Using Web3.js A Beginners Guide - A Voting App',
```
Comment (developerkunal) [inline src/data-tutorials/users.tsx:888]:
```suggestion
    title: 'Interacting with the Celo Blockchain Using Web3.js A Beginners Guide - A Voting App.',
```

**Review by Ishanpathak1 — APPROVED** [review]
Approved @developerkunal

**Pull request merged**